src/biokbase/narrative/jobs/appmanager.py

```diff
@@ -541,7 +547,7 @@ def run_app_bulk(
                 "event_at": datetime.datetime.utcnow().isoformat() + "Z",
                 "cell_id": cell_id,
                 "run_id": run_id,
-                "parent_job_id": batch_submission["parent_job_id"],
+                BATCH_ID_KEY: batch_submission[BATCH_ID_KEY],
```

Comment:
this is currently `parent_job_id` but will get changed to `batch_id` at some point in the near future